docs/tests/integration/test_vulnerability_detector/test_general_settings/index.md

```diff
@@ -28,12 +28,17 @@ process gets started.
 
 ### Generic tests
 
-- **[test_general_settings_enabled](test_general_settings/test_general_settings_enabled.md#test-general-settings-enabled)**:
+- **[test_general_settings_enabled](test_general_settings_enabled.md#test-general-settings-enabled)**:
 Tests that modify the value of `enabled` tag in `ossec.conf` and check the result in `ossec.log`.
 
-- **[test_general_settings_interval](test_general_settings/test_general_settings_interval.md#test-general-settings-interval)**:
+- **[test_general_settings_min_full_scan_interval](test_general_settings_min_full_scan_interval.md#test-general-settings-min-full-scan-interval)**:
+Tests that modify the value of `min_full_scan_interval` tag in `ossec.conf`, set different intervals and checking the result in
+`ossec.log`.
+  
+- **[test_general_settings_interval](test_general_settings_interval.md#test-general-settings-interval)**:
 Tests that modify the value of `interval` tag in `ossec.conf`, set different times and checking the result in
 `ossec.log`.
 
-- **[test_general_settings_run_on_start](test_general_settings/test_general_settings_run_on_start.md#test-general-settings-run-on-start)**:
-Tests that modify the value of `run_on_start` tag in `ossec.conf` and check the result in `ossec.log`.
\ No newline at end of file
+- **[test_general_settings_run_on_start](test_general_settings_run_on_start.md#test-general-settings-run-on-start)**:
+Tests that modify the value of `run_on_start` tag in `ossec.conf` and check the result in `ossec.log`.
```

Comment:
```suggestion
Tests that modify the value of `run_on_start` tag in `ossec.conf` and check the result in the logs file.
```

Same for `ossec.log`

Reply:
Done in: https://github.com/wazuh/wazuh-qa/pull/1369/commits/7438aaa4e055a855a5bd2cbd1fff70d33a42c43f